src/Badge.js

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { space } from 'styled-system'
+import { theme } from '../'
+
+function type (props) {
+  const badgeColors = {
+    'green': {
+      backgroundColor: theme.colors.green,
```

Comment:
Sorry for the delay on this – we shouldn't import the `theme` object into the styled components, but instead get values from `props.theme` which is handled through [React context](https://reactjs.org/docs/context.html) and styled-components' [ThemeProvider](https://www.styled-components.com/docs/advanced#theming) – this allows us better control over the theme and allows things like the design-system's `legacy` switch to control switching between the old and new color palettes